main.ts

```diff
@@ -162,10 +162,13 @@ class Translator {
   visitExternalModuleReferenceExpr(expr: ts.Expression) {
     // TODO: what if this isn't a string literal?
     var moduleName = <ts.StringLiteralExpression>expr;
-    if (!moduleName.text.match(/^\./)) {
-      moduleName.text = 'package:' + moduleName.text;
+    var text = moduleName.text;
+    if (!text.match(/^\./)) {
```

Comment:
this should probably be the same regex.
opt: could evaluate the match once and then use a capture group instead of replace below - but performance is not important here